internal/kgateway/extensions2/plugins/trafficpolicy/gateway_extension.go

```diff
@@ -177,6 +185,8 @@ func TranslateGatewayExtensionBuilder(commoncol *collections.CommonCollections)
 func resolveJwtProviders(
 	krtctx krt.HandlerContext,
 	configMaps krt.Collection[*corev1.ConfigMap],
+	backendResolver backendResolver,
+	objectSource ir.ObjectSource,
```

Comment:
```suggestion
	gwExtObj ir.ObjectSource,
```

assuming this is correct?